### PR TITLE
Update documentation.

### PR DESCRIPTION
**Pull request by nloppi**

### CODE REVIEW

**Comment by pv101** [comment]
Just to confirm that I have tested, and this builds the website OK.

**Comment by FreddieWitherden** [comment]
Please include some notes regarding the fact that schemes can now refer to families.

File: doc/src/developer_guide.rst [hunk 1]
```diff
@@ -162,8 +162,28 @@ inheritance diagram:
                          pyfr.integrators.dual.phys.steppers
     :parts: 1     
                          
-There are five types of pseudo-time `Stepper`_ available in PyFR |release|:                      
-                                           
+
+
+PseudoStepper
+-------
```
Comment (FreddieWitherden) [inline doc/src/developer_guide.rst]: Do we need more _?

Comment (nloppi) [reply doc/src/developer_guide.rst]: True

**Comment by FreddieWitherden** [comment]
Fix line length and we should be good to go.

File: doc/src/developer_guide.rst [hunk 2]
```diff
@@ -224,6 +219,11 @@ The families of schemes follow a naming convention
     :exclude-members: _abc_cache, _abc_negative_cache,
                       _abc_negative_cache_version, _abc_registry
 
+Note that :code:`DualDenseRKPseudoStepper` includes families of
```
Comment (pv101) [inline doc/src/developer_guide.rst]: I don't think we should have the :code: here. We seem to reserve that just for function names.

File: doc/src/developer_guide.rst [hunk 3]
```diff
@@ -224,6 +219,11 @@ The families of schemes follow a naming convention
...
+`PseudoStepper`_ whose coefficients are read from .txt files named thus:
+
+`{scheme name}-s{stage count}-p{temporal order}-sp{optimal spatial polynomial order}.txt`
```
Comment (pv101) [inline doc/src/developer_guide.rst]: We could perhaps ass :code: here to format the file name spec?

Comment (nloppi) [reply doc/src/developer_guide.rst]: Not sure - especially if we use it only with functions? It's now in italics so it pops up from the main text.

File: doc/src/developer_guide.rst (hunk 2 above)
Comment (nloppi) [reply doc/src/developer_guide.rst]: Removed.

File: doc/src/developer_guide.rst (hunk 3 above)
Comment (pv101) [reply doc/src/developer_guide.rst]: OK - although not sure it is currently in italics? But fine without anyway.

**Comment by pv101** [comment]
If @FreddieWitherden is happy with this then I think good to merge. I think we can then mint RC?

**Comment by pv101** [comment]
And you should probably squash into a single commit.

File: doc/src/developer_guide.rst (hunk 3 above)
Comment (nloppi) [reply doc/src/developer_guide.rst]: Ah ok. Not sure how it looks on the website. If I build an html using sphinx it appears in italics. Let's just remove the ` `?

**Comment by nloppi** [comment]
Yes, will do if Freddie approves. Thanks.

**Pull request merged**